packages/bus/README.md

```diff
@@ -164,51 +164,35 @@ await bus.execute(command);
-|-----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `baseUrl` | Application url                                                                                                                                                     |
-| `token`   | Authorization token                                                                                                                                                 |
-| `timeout` | If the request takes longer than `timeout`, the request will be aborted. Default 10000                                                                              |
-| `rate`    | Contain two options: `perMilliseconds` - amount of time to limit concurrent requests; `maxRequests` - max requests to perform concurrently in given amount of time. |
+The `HttpCommandDispatcherConfig` implementation exposes the properties described below:
 
-#### Custom HttpComandDispatcher
+| Option    | Description                                                                                                                                          |
+|-----------|------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `baseUrl` | Application url                                                                                                                                      |
+| `token`   | Authorization token                                                                                                                                  |
+| `timeout` | If the request takes longer than `timeout`, the request will be aborted. Default 10000                                                               |
+| `rate`    | Contain two options: `window` - amount of time to limit concurrent requests; `limit` - max requests to perform concurrently in given amount of time. |
 
-You can implement your own `HttpComandDispatcher`. To do it you should implement`HttpComandDispatcher` interface.
-
-```ts
-import { Command } from '@secbox/core';
-import { HttpComandDispatcher } from '@secbox/bus';
-
-export class CustomHttpDispather implements HttpComandDispatcher {
-  constuctor(/*options*/) {
-    // ...
-  }
-  
-  publick execute<T, R>(command: Command<T, R>): Promise<R> {
-    // your implementation
-  }
-}
-```
-
-#### Executing RPC methods
-
-To execute command `HttpComandDispatcher` exposes a `execute()` method. This method is intended to perform a command to the application and returns an `Promise` with its response.
+To execute command `HttpComandDispatcher` exposes a `execute()` method. This method is intended to perform a command to the application and returns a `Promise` with its response.
```

Comment:
There is no point to repeat the section above, you should focus on the difference between a regular command and HTTP command. So, you can remove the text below entirely.

````suggestion
Then you have to create an instance of `HttpRequest` instead of a custom command, specifying the `url` and `method` in addition to the `payload` that a command accepts by default:

```ts
const command = new HttpCommand({ 
  url: '/api/v1/repeaters', 
  method: 'POST', 
  payload: { name: 'test' } 
});
```

Below you will find a list of parameters that can be used to configure a command:

--- PUT THE TABLE HERE --- 

Once it is done, you can perform a request using `HttpComandDispatcher` as follows:
```ts
const response: { id: string } = await httpDispatcher.execute(command);
```
````